src/App.tsx

```diff
@@ -15,37 +17,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  DEFAULT = '',
+  ALPHABETIC = 'alphabetically',
+  LENGTH = 'length',
+}
+
+interface SortParams {
+  sortField: string;
```

Comment:
You have declared an Enum for this, instead of passing the type as `string` you can use your declared Enum `sortField: SortField`, it is a better approach